### PR TITLE
Piano Analytics Implementation

### PR DESCRIPTION
**Pull request by cami-espinozaq**
### Proposed Changes

It adds Piano Analytics to the App so that it can track the visitor's browsing behaviour.
For now the only views been tracked are the main menu and its tabs (Lernen, Meine Sachen, usw)

### Reason for Changes

Want to get more insight about the user's behaviour in the app.

### Test Coverage

No tests have been added yet.

### Browser Coverage

~Mac~ Linux
 * [x] Chrome 
 * [x] Firefox 
 * [ ] Safari
 
Windows
 * [ ] Chrome 
 * [ ] Firefox 
 * [ ] Edge
 
Chromebook
 * [ ] Chrome
 
iPad
* [ ] Safari

Android Tablet
* [ ] Chrome

### CODE REVIEW

File: yarn.lock
Comment (P3nny) [inline yarn.lock:1]: Is it intenionally that the yarn.lock is commited?

Comment (cami-espinozaq) [reply yarn.lock:1]: it doesn't really matter, I think. I could delete it though, if you want

Comment (jh0ker) [reply yarn.lock:1]: It's needed so the deployment uses the exact same dependencies as you use in development.

**Review by P3nny — APPROVED** [review]

**Pull request merged**